### ISSUE TITLE
Mobile interactions for RMRK2

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#5955]
@JustLuuuu mentioned that KodaDot doesn't work at some cases from NovaWallet. 

I would like to see if someone from @kodadot/qa-guild could go through KodaDot available currently on https://beta.kodadot.xyz and check if they spot some busg i.e. for interactions like LIST/BUY on RMRK2 primarily or some mobile design glitches.

Happy to pay for reporting bug in $KSM!

**Comment by prury** [comment]
> 



> @JustLuuuu mentioned that KodaDot doesn't work at some cases from NovaWallet.
> 
> I would like to see if someone from @kodadot/qa-guild could go through KodaDot available currently on https://beta.kodadot.xyz and check if they spot some busg i.e. for interactions like LIST/BUY on RMRK2 primarily or some mobile design glitches.
> 
> Happy to pay for reporting bug in $KSM!

i can do it on Monday if that's not a problem

**Comment by yangwao** [comment]
> i can do it on Monday if that's not a problem

any finds?

**Comment by prury** [comment]
> > i can do it on Monday if that's not a problem
> 
> any finds?

tried to message you on telegram saying that i had no time to check it yesterday, but it seems that the telegram messages never really reached you. Will dive into it right now

Only thing that i was able to identify yesterday is that, if you want to change networks when using a mobile, you have to do it manually via browser, the network change button just disappears when the window is resized to a mobile screen. This could be one of difficulties people are having when trying to reach RMRK2 collections and items. 

Will deep dive into the other issues right now

**Comment by yangwao** [comment]
- https://github.com/kodadot/nft-gallery/issues/5877

**Comment by prury** [comment]
- i was not able to add a NFT to a collection(collections go trough without a problem), it always says that my balance is low even if i have >$1 dollar on my account

<img src="https://github.com/kodadot/nft-gallery/assets/36627808/43692285-4b7c-48a8-9d0d-3828aac437a7" height="500" height="100">

**Comment by yangwao** [comment]
> you have to do it manually via browser, the network change button just disappears when the window is resized to a mobile screen, maybe is intended.

this is worth issue and checking

**Comment by yangwao** [comment]
> tried to message you on telegram saying that i had no time to check it yesterday

uh, let me add u to attention folder

**Comment by prury** [comment]
When trying to create collections/nfts, two options appears, classic and simple, when you click on simple, the following occurs:

<img src="https://github.com/kodadot/nft-gallery/assets/36627808/614ad02b-c296-40a2-a67a-1b25455d19d9" height="500" height="100">
<img src="https://github.com/kodadot/nft-gallery/assets/36627808/c6de78ad-ceb6-4a0c-bb3f-5343b709e52e" height="500" height="100">

**Comment by prury** [comment]
Even tough I'm using the same account on both mobile phone and android emulator (JSON acc), it wont show the collection I've created on one of them. 
<img src="https://github.com/kodadot/nft-gallery/assets/36627808/315c8bf1-044a-46cf-b498-75702029855e" height="500" height="100"><img src="https://github.com/kodadot/nft-gallery/assets/36627808/ca0af7af-8653-4cf9-8f83-e3abd7ba4468" height="500" height="100">

**Comment by prury** [comment]
@yangwao lmk if i have to create an issue for each one of those problems or if i should just throw them here

**Comment by yangwao** [comment]
> if i should just throw them here

yes please, make separate issues best, sense it's two different problems

**Comment by prury** [comment]
> > if i should just throw them here
> 
> yes please, make separate issues best, sense it's two different problems

cool, can u add Novawallet in the wallet dropdown template for bug reports?

**Comment by prury** [comment]
> > you have to do it manually via browser, the network change button just disappears when the window is resized to a mobile screen, maybe is intended.
> 
> this is worth issue and checking

Checked again and if you chose the desired network in the main page it changes automatically: 
![image](https://github.com/kodadot/nft-gallery/assets/36627808/e6ad133d-e771-4826-a4d7-c66bcd7cc37e)

if you think we need the button itself (for ppl that are navigating outside this page) I'll open a issue

**Comment by exezbcz** [comment]
@prury nope, this behavior is not meant to be a network switch - its a shortcut to /explore/network you choose.
yes, please open an issue 👀

**Comment by prury** [comment]
> @prury nope, this behavior is not meant to be a network switch - its a shortcut to /explore/network you choose. yes, please open an issue 👀

done: #6029
